### PR TITLE
Qc checks

### PR DESCRIPTION
**Pull request by hkir-dev**
Related with the issue #32 

Checks:

- if IDs in the template are all unique
- Only one of 'Equivalent' or 'Subclass part of' have value, but not both
- All entities should exist in the structure graph and their label's should match

Sample output:
```
=== Single Mapping Checks :
http://purl.obolibrary.org/obo/MBA_190 has both Equivalent and SubClassOf

=== Unique Id Checks :
http://purl.obolibrary.org/obo/MBA_10687 exists in multiple lines: 73, 1260 with different mappings.
http://purl.obolibrary.org/obo/MBA_10689 exists in multiple lines: 75, 1261, 1262 with different mappings.
http://purl.obolibrary.org/obo/MBA_10690 exists in multiple lines: 76, 1263 with different mappings.
http://purl.obolibrary.org/obo/MBA_10691 exists in multiple lines: 77, 1264 with different mappings.
http://purl.obolibrary.org/obo/MBA_10692 exists in multiple lines: 78, 1265 with different mappings.
http://purl.obolibrary.org/obo/MBA_10705 exists in multiple lines: 96, 1272 with different mappings.
http://purl.obolibrary.org/obo/MBA_10706 exists in multiple lines: 97, 1273 with different mappings.
http://purl.obolibrary.org/obo/MBA_10707 exists in multiple lines: 98, 1274 with different mappings.
http://purl.obolibrary.org/obo/MBA_118 exists in multiple lines: 179, 180, 1221 with different mappings.
http://purl.obolibrary.org/obo/MBA_190 exists in multiple lines: 279, 1244 with different mappings.
http://purl.obolibrary.org/obo/MBA_30 exists in multiple lines: 398, 1220 with different mappings.
http://purl.obolibrary.org/obo/MBA_401 exists in multiple lines: 1319, 1336 with different mappings.

=== Structure Graph Compatibility :
http://purl.obolibrary.org/obo/MBA_181 label is 'nucleus of reunions' in template, but 'Nucleus of reuniens' in the structure graph.
http://purl.obolibrary.org/obo/MBA_181 label is 'nucleus of reunions' in template, but 'Nucleus of reuniens' in the structure graph.
http://purl.obolibrary.org/obo/MBA_428 label is 'medial corticohypothalmic tract' in template, but 'medial corticohypothalamic tract' in the structure graph.
http://purl.obolibrary.org/obo/MBA_500 label is 'somatomotor area' in template, but 'Somatomotor areas' in the structure graph.
http://purl.obolibrary.org/obo/MBA_1057 label is 'gustatory cortex' in template, but 'Gustatory areas' in the structure graph.
http://purl.obolibrary.org/obo/MBA_10689 label is 'paraflocculus, Purkinje layer' in template, but 'Paraflocculus, molecular layer' in the structure graph.
```

### CODE REVIEW

**Comment by shawntanzk** [comment]
nice, this check is brilliant :D will fix the issues. You can tell which ones I manually inputted from this LOL

File: .github/workflows/mapping_check.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
```
Comment (shawntanzk) [inline .github/workflows/mapping_check.yaml:6]: Could I suggest the following instead: 

```
  pull_request:
    branches: [ master ]
    paths:
 ```

Comment (hkir-dev) [reply .github/workflows/mapping_check.yaml]: updated

**Comment by shawntanzk** [comment]
Fixed everything and updated all, will merge, looks good, thanks for this really helps with picking up silly errors

**Pull request merged**